src/renderer/components/BackgroundMaps/BackgroundMapInfo.js

```diff
@@ -0,0 +1,198 @@
+  const { formatMessage: t } = useIntl()
+
+  const [mapStyle, setMapStyle] = useBackgroundMapStore()
+
+  const classes = useStyles()
+
+  const isCurrentMap = mapStyle?.id === map.id
+
+  return (
+    <Fade in timeout={0}>
+      <Paper
+        style={{
+          flex: 1,
+          width: '100%',
+          height: '100%',
+          padding: !map ? 40 : 0
+        }}
+      >
+        {!map ? (
+          <Loading />
+        ) : (
+          <>
+            <MapInfo
+              name={typeof map.name === 'string' ? map.name : t(map.name)}
+              id={map.id}
+              unsetMapValue={unsetMapValue}
+              url={map.url}
+              isDefault={map.isDefault}
+            />
+            {/* Text */}
+            <div className={classes.paddedContainer}>
+              <Typography variant='subtitle2' style={{ fontSize: 18 }}>
+                {typeof map.name === 'string' ? map.name : t(map.name)}
+              </Typography>
+              {!map.isDefault ? (
+                <Typography variant='body1'>{`${Math.round(
+                  convertKbToMb(map.bytesStored)
+                )} ${t(m.mb)}`}</Typography>
+              ) : null}
+              <Button
+                variant='outlined'
+                onClick={() => setMapStyle(map)}
+                className={`${classes.paddedButton} ${isCurrentMap &&
+                  classes.iconButton}`}
+                disabled={map.id === mapStyle?.id}
+              >
+                <Typography
+                  style={{ textTransform: 'none' }}
+                  variant='subtitle2'
+                >
+                  {isCurrentMap ? t(m.currentMap) : t(m.useMap)}
+                </Typography>
+                {isCurrentMap ? <CheckIcon className={classes.icon} /> : null}
+              </Button>
+            </div>
+          </>
+        )}
+      </Paper>
+    </Fade>
+  )
+}
+
+/**
+ * @typedef MapInfoProps
+ * @prop {string|null|undefined} name
+ * @prop {string} id
+ * @prop {()=>void} unsetMapValue
+ * @prop {string} url
+ * @prop {boolean | undefined} isDefault
+ */
+
+/** @param {MapInfoProps} props */
+const MapInfo = ({ name, id, isDefault, unsetMapValue, url }) => {
+  const classes = useStyles()
+
+  const { formatMessage: t } = useIntl()
+
+  const mutation = useMapServerMutation('delete', `/styles/${id}`)
+
+  async function deleteMap () {
+    await mutation.mutateAsync(null) // tc complains if no arg is passed...
+  }
+
+  return (
+    <>
+      {/* Banner */}
+      <Paper className={classes.banner}>
+        <Typography variant='h5'>{name}</Typography>
+
+        <div>
+          {!isDefault && (
+            <Button variant='outlined' onClick={() => deleteMap()}>
+              <DeleteIcon className={classes.icon} />
+              <Typography style={{ textTransform: 'none' }} variant='subtitle2'>
+                {t(m.deleteStyle)}
+              </Typography>
+            </Button>
+          )}
+        </div>
```

Comment:
this is currently not working, but I believe this is a map server thing, so just remove it for now (and the user will not be able to delete a map)

Reply:
this is working fine for me, lets check together later whats going on

Reply:
(I added a little check so it will set the the default map if you delete the current selected one)